docs/en/ufs/NFS.md

```diff
@@ -54,8 +58,8 @@ Run a simple example program:
 ./bin/alluxio runTests
 ```
 
-Visit your NFS volume to verify the files and directories created
-by Alluxio exist. For this test, you should see files named like:
+Visit your NFS volume at `/mtn/nfs` to verify the files and directories created by Alluxio exist.
+For this test, you should see files named like:
```

Comment:
Remove the like. Or change to "file with the naming convention". Might just be me, but "like" seems too informal for documentation.

Reply:
agreed